### PR TITLE
chore: create dependency bump template for releases

### PR DESCRIPTION
**Pull request by chair28980**
Creates dependency bump template for releases.

### CODE REVIEW

**Comment by chair28980** [comment]
@Ivansete-status are there any other labels that should be included for this issue template?

**Comment by Ivansete-status** [comment]
@chair28980, kindly add the next to the template:

```
- [ ] dnsclient.nim ( update to the latest tag version )
- [ ] nim-bearssl
- [ ] nimbus-build-system
- [ ] nim-chronicles
- [ ] nim-chronos
- [ ] nim-confutils
- [ ] nimcrypto
- [ ] nim-dnsdisc
- [ ] nim-eth
- [ ] nim-faststreams
- [ ] nim-http-utils
- [ ] nim-json-rpc
- [ ] nim-json-serialization
- [ ] nim-libbacktrace
- [ ] nim-libp2p
- [ ] nim-metrics
- [ ] nim-nat-traversal
- [ ] nim-presto
- [ ] nim-regex ( update to the latest tag version )
- [ ] nim-secp256k1
- [ ] nim-serialization
- [ ] nim-sqlite3-abi ( update to the latest tag version )
- [ ] nim-stew
- [ ] nim-stint
- [ ] nim-taskpools
- [ ] nim-testutils
- [ ] nim-toml-serialization
- [ ] nim-unicodedb
- [ ] nim-unittest2
- [ ] nim-web3
- [ ] nim-websock
- [ ] nim-zlib
- [ ] zerokit ( this should be kept in version `v0.3.4` )
```
Cheers

File: .github/ISSUE_TEMPLATE/bump_dependancies.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+name: Bump dependancies
+about: Bump vender dependencies for release
```
Comment (SionoiS) [inline .github/ISSUE_TEMPLATE/bump_dependancies.md:3]:
```suggestion
about: Bump vendor dependencies for release
```

File: .github/ISSUE_TEMPLATE/bump_dependancies.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+title: 'Bump vender dependencies for release 0.0.0'
```
Comment (SionoiS) [inline .github/ISSUE_TEMPLATE/bump_dependancies.md:4]:
```suggestion
title: 'Bump vendor dependencies for release 0.0.0'
```

File: .github/ISSUE_TEMPLATE/bump_dependancies.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+name: Bump dependancies
```
Comment (SionoiS) [inline .github/ISSUE_TEMPLATE/bump_dependancies.md:2]:
```suggestion
name: Bump dependencies
```

File: .github/ISSUE_TEMPLATE/bump_dependancies.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+about: Bump vender dependencies for release
+title: 'Bump vender dependencies for release 0.0.0'
+labels: enhancement
```
Comment (SionoiS) [inline .github/ISSUE_TEMPLATE/bump_dependancies.md:5]: Was there not a talk about using a `dependency` label?

https://github.com/waku-org/pm/pull/91

**Review by SionoiS — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bump_dependancies.md (hunk 4 above)
Comment (chair28980) [reply .github/ISSUE_TEMPLATE/bump_dependancies.md]: Yes indeed, looks like `dependencies` was settled on, I updated the template as such.

File: .github/ISSUE_TEMPLATE/bump_dependancies.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+- [ ] nim-stint
+- [ ] nim-taskpools
+- [ ] nim-testutils
+- [ ] nim-toml-serialization
+- [ ] nim-unicodedb
+- [ ] nim-unittest2
+- [ ] nim-web3
+- [ ] nim-websock
+- [ ] nim-zlib
+- [ ] zerokit ( this should be kept in version `v0.3.4` )
```
Comment (Ivansete-status) [inline .github/ISSUE_TEMPLATE/bump_dependancies.md:47]: Tiny nitpick. Add an empty line at the end :P

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks for it!

The `dependencies` label is correct as per the next comment: https://github.com/waku-org/pm/pull/91#issuecomment-1728786822

**Review by SionoiS — APPROVED** [review]

**Pull request merged**